pkg/apis/componentconfig/v1/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=package
+// +k8s:conversion-gen=k8s.io/apiserver/pkg/admission/plugin/webhook/config/apis/webhookadmission
+// +k8s:defaulter-gen=TypeMeta
+
+// Package v1 is the v1 version of the API.
+// +groupName=apiserver.config.k8s.io
+package v1
```

Comment:
This is in the wrong package, as @sttts pointed out. No more API types should be added here, and this is a catch-all directory now which we're moving away from. I think `pkg/kubeapiserver/apis/kubeapiserverconfig` makes sense in case it's kube-apiserver-specific or `k8s.io/apiserver/apis/apiserverconfig` if it's generic for apiservers.

Reply:
> I think `pkg/kubeapiserver/apis/kubeapiserverconfig` makes sense in case it's kube-apiserver-specific

`pkg/kubeapiserver/apis/kubeapiserverconfig` doesn't exist, should I create it then?

> or `k8s.io/apiserver/apis/apiserverconfig` if it's generic for apiservers

`k8s.io/apiserver/apis/apiserverconfig` doesn't exist either. Did you mean `vendor/k8s.io/apiserver/pkg/apis/apiserverconfig`? In this case, `apiserverconfig` sub directory also should be created.